shell/conf/benchmarks_defaults.conf

```diff
@@ -20,25 +20,14 @@
 # Logs
 ALOJA_FORCE_COLORS="1" #Force colors on screen even when writing log files
 
-# Data sizes and default scale factor
-if [ "$BENCH_SCALE_FACTOR" ] ; then
-   BENCH_DATA_SIZE="$(( BENCH_SCALE_FACTOR * 1000000000 ))" #in GB
-else
-   if [ ! "$BENCH_DATA_SIZE" ] ; then
-     # Check if TPC scale factor has been specified
-     if [[ "$TPCH_USE_LOCAL_FACTOR" > 0 ]]; then 
-       BENCH_DATA_SIZE="$(( TPCH_USE_LOCAL_FACTOR *  1000000000 ))" # if set, it is in GBs
-     else
-       if [ "$TPCH_SCALE_FACTOR" ] ; then
-         BENCH_DATA_SIZE="$(( TPCH_SCALE_FACTOR *  1000000000 ))" # if set, it is in GBs
-       else
-         BENCH_DATA_SIZE="100000000" #100MB, in bytes #1000000000000 1TB
-       fi
-     fi
-   fi
-fi
+BYTES_IN_MB=1000000
+BYTES_IN_GB=1000000000
+BYTES_IN_TB=1000000000000
 
-BENCH_SCALE_FACTOR="$(( BENCH_DATA_SIZE / 1000000000 ))" #in GB
+if [ ! "$BENCH_DATA_SIZE" ] ; then
+  BENCH_DATA_SIZE="100MB" #100MB minimum size
+  set_benchmark_data_sizes "$(get_bench_data_size_bytes "$BENCH_DATA_SIZE")"
```

Comment:
This is not a very user-friendly API. Why should I have to use two functions to do one thing (set the data size)?

An easier to use API would be `set_benchmark_data_size 100MB`, then let the implementation worry about performing any necessary conversions and saving the result to the global variable.

This is also nasty because `BENCH_DATA_SIZE` gets written to twice, on the first line which is obvious, but then again on the second line, inside `set_benchmark_data_sizes`. From the perspective of reading just these lines of code, what would you expect the value of `BENCH_DATA_SIZE` to be? I think most people would say `100MB` (and I wouldn't blame them!).